admin-auth/src/main/kotlin/com/epam/drill/admin/auth/service/impl/OAuthMapperImpl.kt

```diff
@@ -55,10 +55,15 @@ class OAuthMapperImpl(oauth2Config: OAuth2Config) : OAuthMapper {
     }
 
     private fun mapRole(roleNames: List<String>?): Role {
-        roleNames?.forEach {
-            when (it.lowercase()) {
-                roleMapping.user.lowercase() -> return Role.USER
-                roleMapping.admin.lowercase() -> return Role.ADMIN
+        val userRoles = roleMapping.user.split(",").map { it.trim().lowercase() }
```

Comment:
I think `application.conf` syntax supports lists in the form of 
```
value = [ "foo", "bar", "baz"]
```
Should we just use those instead of comma-separated string?